packages/platform/constants/timezones.ts

```diff
@@ -0,0 +1,337 @@
+export type TimeZone = (typeof TIMEZONES)[number];
+
+const TIMEZONES = [
```

Comment:
Was this taken from the main app?

Reply:
Yes. “packages/ui/components/form/timezone-select/TimezoneSelect.tsx” invokes "cityTimezones.useQuery" which returns "data" which is then passed to "addCitiesToDropdown" which returns "cityTimezones" which are passed to "timezones" of the dropdown. I logged "cityTimezones" and these are values that are possible in the dropdown when selecting / writing timezones.